OpenRA.Mods.Common/Activities/Air/FlyAttack.cs

```diff
@@ -120,6 +122,19 @@ public override bool Tick(Actor self)
 			// and resume the activity after reloading if AbortOnResupply is set to 'false'
 			if (rearmable != null && !useLastVisibleTarget && attackAircraft.Armaments.All(x => x.IsTraitPaused || !x.Weapon.IsValidAgainst(target, self.World, self)))
 			{
+				// Units may want to continue moving to the ordered location instead of resupplying
+				if (source == AttackSource.AttackMove && !attackAircraft.Info.ResupplyDuringAttackMove)
+					return true;
+
+				if (attackAircraft.Info.AbortOnResupply)
+				{
+					// HACK: FlyAttack may be a child activity (e.g. of AttackMoveActivity)
```

Comment:
Minor general style nit: If there are more comment lines below that don't continue the sentence, please always use full stops.